### ISSUE TITLE
Upgrade 6.0.25 to 6.0.26 returns cannot find module 'less-loader'

### ISSUE THREAD

**Issue by justinhartman** [JeffreyWay/laravel-mix#3051]
-   Laravel Mix Version: `6.0.26` (`npm list --depth=0`)
```text
├── axios@0.21.1
├── bootstrap@4.6.0
├── cross-env@7.0.3
├── dinero.js@1.9.0
├── flag-icon-css@3.5.0
├── git-changelog@2.0.0
├── grunt@1.4.1
├── jquery@3.6.0
├── laravel-mix@6.0.26
├── laravel-vue-pagination@2.3.1
├── lodash@4.17.21
├── moment@2.29.1
├── popper.js@1.16.1
├── postcss@8.3.6
├── promise@8.1.0
├── resolve-url-loader@4.0.0
├── rtlcss@3.3.0
├── sass-loader@12.1.0
├── sass@1.32.8
├── sweetalert2@10.16.9
├── urijs@1.19.7
├── v-calendar@2.3.2
├── vue-date-pick@1.4.1
├── vue-loader@16.3.3
├── vue-template-compiler@2.6.14
├── vue@2.6.14
└── vue2-editor@2.10.2
```
-   Node Version (`node -v`): `v12.14.1`
-   NPM Version (`npm -v`): `7.19.1`
-   OS: `macOS 10.15.7`

### Description:

I had version `6.0.25` installed and ran an `npm upgrade` and trying to do a `npm run dev` (or any run for that matter) returns the following error:

```console
$ npm run dev

> dev
> npm run development


> development
> cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --config=node_modules/laravel-mix/setup/webpack.config.js

[webpack-cli] Error: Cannot find module 'less-loader'
Require stack:
- ./node_modules/laravel-mix/src/components/CssWebpackConfig.js
- ./node_modules/laravel-mix/src/components/Preprocessor.js
- ./node_modules/laravel-mix/src/components/Less.js
- ./node_modules/laravel-mix/src/components/ComponentRegistrar.js
- ./node_modules/laravel-mix/src/Mix.js
- ./node_modules/laravel-mix/setup/webpack.config.js
- ./node_modules/webpack-cli/lib/webpack-cli.js
- ./node_modules/webpack-cli/lib/bootstrap.js
- ./node_modules/webpack-cli/bin/cli.js
- ./node_modules/webpack/bin/webpack.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
    at Function.resolve (./node_modules/v8-compile-cache/v8-compile-cache.js:164:23)
    at CssWebpackConfig.webpackRules (./node_modules/laravel-mix/src/components/CssWebpackConfig.js:63:43)
    at ComponentRegistrar.applyRules (./node_modules/laravel-mix/src/components/ComponentRegistrar.js:191:23)
    at ./node_modules/laravel-mix/src/components/ComponentRegistrar.js:98:48
    at ./node_modules/laravel-mix/src/Dispatcher.js:37:62
    at Array.map (<anonymous>)
    at Dispatcher.fire (./node_modules/laravel-mix/src/Dispatcher.js:37:47)
    at Mix.dispatch (./node_modules/laravel-mix/src/Mix.js:246:32)
    at WebpackConfig.buildRules (./node_modules/laravel-mix/src/builder/WebpackConfig.js:165:24) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    './node_modules/laravel-mix/src/components/CssWebpackConfig.js',
    './node_modules/laravel-mix/src/components/Preprocessor.js',
    './node_modules/laravel-mix/src/components/Less.js',
    './node_modules/laravel-mix/src/components/ComponentRegistrar.js',
    './node_modules/laravel-mix/src/Mix.js',
    './node_modules/laravel-mix/setup/webpack.config.js',
    './node_modules/webpack-cli/lib/webpack-cli.js',
    './node_modules/webpack-cli/lib/bootstrap.js',
    './node_modules/webpack-cli/bin/cli.js',
    './node_modules/webpack/bin/webpack.js'
  ]
}
```

So I then did added the package it was complaining about with `npm add less-loader` and then got a new error:

```console
$ npm run dev

> dev
> npm run development


> development
> cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --config=node_modules/laravel-mix/setup/webpack.config.js

[webpack-cli] Error: Cannot find module 'stylus-loader'
Require stack:
- ./node_modules/laravel-mix/src/components/CssWebpackConfig.js
- ./node_modules/laravel-mix/src/components/Preprocessor.js
- ./node_modules/laravel-mix/src/components/Less.js
- ./node_modules/laravel-mix/src/components/ComponentRegistrar.js
- ./node_modules/laravel-mix/src/Mix.js
- ./node_modules/laravel-mix/setup/webpack.config.js
- ./node_modules/webpack-cli/lib/webpack-cli.js
- ./node_modules/webpack-cli/lib/bootstrap.js
- ./node_modules/webpack-cli/bin/cli.js
- ./node_modules/webpack/bin/webpack.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
    at Function.resolve (./node_modules/v8-compile-cache/v8-compile-cache.js:164:23)
    at CssWebpackConfig.webpackRules (./node_modules/laravel-mix/src/components/CssWebpackConfig.js:69:43)
    at ComponentRegistrar.applyRules (./node_modules/laravel-mix/src/components/ComponentRegistrar.js:191:23)
    at ./node_modules/laravel-mix/src/components/ComponentRegistrar.js:98:48
    at ./node_modules/laravel-mix/src/Dispatcher.js:37:62
    at Array.map (<anonymous>)
    at Dispatcher.fire (./node_modules/laravel-mix/src/Dispatcher.js:37:47)
    at Mix.dispatch (./node_modules/laravel-mix/src/Mix.js:246:32)
    at WebpackConfig.buildRules (./node_modules/laravel-mix/src/builder/WebpackConfig.js:165:24) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    './node_modules/laravel-mix/src/components/CssWebpackConfig.js',
    './node_modules/laravel-mix/src/components/Preprocessor.js',
    './node_modules/laravel-mix/src/components/Less.js',
    './node_modules/laravel-mix/src/components/ComponentRegistrar.js',
    './node_modules/laravel-mix/src/Mix.js',
    './node_modules/laravel-mix/setup/webpack.config.js',
    './node_modules/webpack-cli/lib/webpack-cli.js',
    './node_modules/webpack-cli/lib/bootstrap.js',
    './node_modules/webpack-cli/bin/cli.js',
    './node_modules/webpack/bin/webpack.js'
  ]
}
```

I gave up this plight and reverted my mix version back to 6.0.25 and ran the following:

```console
$ npm run dev

> dev
> npm run development


> development
> cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --config=node_modules/laravel-mix/setup/webpack.config.js


   Laravel Mix v6.0.25


✔ Compiled Successfully in 59131ms
┌───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┬───────────┐
...
└───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┴───────────┘
webpack compiled successfully
```

### Steps To Reproduce:

1. Run with this package.json file

```json
{
  "private": true,
  "scripts": {
    "dev": "npm run development",
    "development": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --config=node_modules/laravel-mix/setup/webpack.config.js",
    "watch": "npm run development -- --watch",
    "hot": "cross-env NODE_ENV=development node_modules/webpack-dev-server/bin/webpack-dev-server.js --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
    "prod": "npm run production",
    "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-devtool --no-stats --env production --config=node_modules/laravel-mix/setup/webpack.config.js"
  },
  "dependencies": {
    "axios": "^0.21",
    "bootstrap": "^4.6.0",
    "cross-env": "^7.0.3",
    "dinero.js": "^1.8.1",
    "flag-icon-css": "^3.5.0",
    "jquery": "^3.6.0",
    "laravel-mix": "6.0.26",
    "laravel-vue-pagination": "^2.3.1",
    "lodash": "^4.17.20",
    "moment": "^2.29.1",
    "popper.js": "^1.16.1",
    "promise": "^8.1.0",
    "resolve-url-loader": "4.0.0",
    "rtlcss": "^3.3.0",
    "sass": "1.32.8",
    "sass-loader": "^12.1.0",
    "sweetalert2": "^10.16.9",
    "urijs": "^1.19.7",
    "v-calendar": "^2.3.1",
    "vue": "^2.6.12",
    "vue-date-pick": "^1.4.1",
    "vue-template-compiler": "^2.6.12",
    "vue2-editor": "^2.10.2"
  },
  "devDependencies": {
    "git-changelog": "^2.0.0",
    "grunt": "^1.4.0",
    "postcss": "^8.3.2",
    "vue-loader": "^16.3.0"
  }
}
```

2. Revert Laravel Mix to `6.0.25` and the error goes away and builds as expected.

**Comment by thecrypticace** [comment]
We fixed this in 6.0.27. Can you bump your version of Mix please?

**Comment by justinhartman** [comment]
@thecrypticace I can confirm 6.0.27 resolves the issue. Sorry, I didn't see you had released .27 as mine was only reflecting .26 as the current version else I wouldn't have opened the issue. Many thanks!
